content/material_and_questions.md

```diff
@@ -44,4 +52,28 @@ In the below table, any minimum or maximum variation between material years are
 | Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
 | Situation       |     2/0 |     4/0 | SIT                  |
 
+### Question Type Minimums
```

Comment:
This rule needs to follow rules about quizzes. At this point in the document architecture, there's not yet a defined notion of a quiz.

Reply:
I'd love ideas on organization. Minimums/Maximums need to be "near" Question Type defs....but yes, also need to come "after" the concept of a quiz has been defined.